pkg/api/const.go

```diff
@@ -414,6 +419,8 @@ const (
 	DefaultKubernetesAgentSubnetVMSS = "10.248.0.0/13"
 	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
 	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
+	// DefaultKubernetesClusterSubnetIPv6 specifies the IPv6 default subnet for pods.
+	DefaultKubernetesClusterSubnetIPv6 = "ace:cab:deca::/8"
```

Comment:
can we use one of `fec0::/7` addresses instead?

Reply:
@khenidak tried using that subnet and creating the cluster. The controller panics and fails to start because the cidr is too big - 

```
E0605 21:25:49.614408       1 controllermanager.go:520] Error starting "nodeipam"
F0605 21:25:49.614635       1 controllermanager.go:238] error starting controllers: New CIDR set failed; the node CIDR size is too big
goroutine 119 [running]:
```